vertical-pod-autoscaler/pkg/utils/test/test_utils.go

```diff
@@ -32,6 +32,7 @@ import (
 
 // BuildTestPod creates a pod with specified resources.
 func BuildTestPod(name, containerName, cpu, mem string, creatorObjectMeta *metav1.ObjectMeta, creatorTypeMeta *metav1.TypeMeta) *apiv1.Pod {
+	startTime := metav1.Now()
```

Comment:
Please use a fixed date instead of Now()

Reply:
Done.